src/views/pages/product/single.twig

```diff
@@ -372,7 +390,9 @@
                         {% endif %}
                         {# Add to cart #}
                         <salla-add-product-button
-                                quick-buy
+                                {% if not product.has_preorder_campaign %}
+                                    quick-buy
+                                {% endif %}
```

Comment:
please support from backend: `product.can_quick_buy`
and change this to be `{{product.can_quick_buy ? 'quick-buy' : ''}}`
same other props